src/index.js

```diff
@@ -57,16 +57,18 @@ const exported = {
      * used, must be set before `prewarm()` is called to have an effect.
      *
      * By default, the lifecycle of these resources is managed automatically, and they are
-     * lazily initialized when a Map is first created. By invoking `prewarm()`, these
-     * resources will be created ahead of time, and will not be cleared when the last Map
-     * is removed from the page. This allows them to be re-used by new Map instances that
+     * lazily initialized when a `Map` is first created. By invoking `prewarm()`, these
```

Comment:
Nit: Suggested simplification of wording:

> Invoking `prewarm()` creates these resources ahead of time, and they are not cleared when the last `Map` is removed from the page.

Reply:
```suggestion
     * lazily initialized when a `Map` is first created. Invoking `prewarm()` creates these
```

Reply:
👍  I think maybe it spilled onto subsequent lines, which is why I didn't do this in suggestion form… multi-line suggestions aren't yet a thing, are they?

Reply:
I don't think so, and you may be right! I was just trying to make this as easy as possible so that we can get it merged and in v2.5.